apps/fluent-tester/src/TestComponents/VibrancyView/VibrancyViewTest.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import { Alert, type ViewProps } from 'react-native';
+import { View, StyleSheet } from 'react-native-macos';
+
+import { ButtonV1 as Button, Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import type { Material, BlendingMode, State } from '@fluentui-react-native/vibrancy-view';
+import { VibrancyView } from '@fluentui-react-native/vibrancy-view';
+
+import { VIBRANCYVIEW_TESTPAGE } from '../../../../E2E/src/VibrancyView/consts';
+import { stackStyle, commonTestStyles as commonStyles } from '../Common/styles';
+import type { TestSection, PlatformStatus } from '../Test';
+import { Test } from '../Test';
+import { MenuPicker } from '../Common/MenuPicker';
+
+const styles = StyleSheet.create({
+  spacing: {
+    gap: 8,
+  },
+  box: {
+    width: 100,
+    height: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  bigBox: {
+    width: 200,
+    height: 200,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  biggerBox: {
+    backgroundColor: 'yellow',
+    width: 300,
+    height: 300,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  Rectangle1: {
+    backgroundColor: 'yellow',
+    width: 300,
+    height: 300,
+  },
+  Rectangle2: {
+    position: 'absolute',
+    top: 50,
+    width: 400,
+    height: 200,
+  },
+  centeredBox: {
+    backgroundColor: 'green',
+    width: 100,
+    height: 100,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  borderProps: {
+    backgroundColor: 'blue',
+    borderColor: 'red',
+    borderWidth: 1,
+    borderRadius: 8,
+  },
+});
+
+const VibrancyViewDefault: React.FunctionComponent = () => {
+  const viewProps: ViewProps = {
+    focusable: true,
+    style: [styles.box, styles.borderProps],
+  };
+  return (
+    <Stack style={stackStyle}>
+      <View style={[commonStyles.root, styles.spacing]}>
```

Comment:
nit: should be a const defined outside of the component or memoized

Reply:
Apply to other Views with inline style arrays as well

Reply:
I didn't think that applied to array styles? But sure

Reply:
The object passed in would be regenerated on every render (since the array object would be newly created), causing a wasted render assuming the style/other props didn't change